app/Services/MaintenanceWindowService.php

```
@@ -0,0 +1,368 @@
+        }
+
+        return null;
+    }
+
+    /**
+     * Clean up expired one-time maintenance windows.
+     */
+    public function cleanupExpiredWindows(): int
+    {
+        $cleaned = 0;
+        $now = Carbon::now();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$now'. (undefined)

(UnusedLocalVariable)